tests/Resources/ti.modulesdk920.test.js

```diff
@@ -0,0 +1,254 @@
+	});
+
+	it('.dateValue', () => {
+		const currentDate = new Date();
+		should(tiModule.dateValue).be.equalOneOf([ null, undefined ]);
+		tiModule.dateValue = currentDate;
+		should(tiModule.dateValue).be.eql(currentDate);
+		tiModule.dateValue = null;
+		should(tiModule.dateValue).be.equalOneOf([ null, undefined ]);
+	});
+
+	it('.dateArray', () => {
+		const currentDate = new Date();
+		const oldDate = new Date(2020, 0, 31);
+		should(tiModule.dateArray).be.equalOneOf([ null, undefined ]);
+		tiModule.dateArray = [ currentDate, oldDate ];
+		should(tiModule.dateArray).be.eql([ currentDate, oldDate ]);
+		tiModule.dateArray = [];
+		should(tiModule.dateArray).be.eql([]);
+		tiModule.dateArray = null;
+		should(tiModule.dateArray).be.equalOneOf([ null, undefined ]);
+	});
+
+	it('.dictionaryValue', () => {
+		should(tiModule.dictionaryValue).be.equalOneOf([ null, undefined ]);
+		tiModule.dictionaryValue = {};
+		should(tiModule.dictionaryValue).be.eql({});
+		tiModule.dictionaryValue = null;
+		should(tiModule.dictionaryValue).be.equalOneOf([ null, undefined ]);
+		tiModule.dictionaryValue = DICTIONARY_TEST_DATA;
+		should(tiModule.dictionaryValue).be.eql(DICTIONARY_TEST_DATA);
+	});
+
+	it('.dictionaryArray', () => {
+		should(tiModule.dictionaryArray).be.equalOneOf([ null, undefined ]);
+		tiModule.dictionaryArray = [];
+		should(tiModule.dictionaryArray).be.eql([]);
+		tiModule.dictionaryArray = null;
+		should(tiModule.dictionaryArray).be.equalOneOf([ null, undefined ]);
+		tiModule.dictionaryArray = [ {} ];
+		should(tiModule.dictionaryArray).be.eql([ {} ]);
+		tiModule.dictionaryArray = [ DICTIONARY_TEST_DATA, { foo: 'bar' } ];
+		should(tiModule.dictionaryArray).be.eql([ DICTIONARY_TEST_DATA, { foo: 'bar' } ]);
+	});
+
+	it('.callback', () => {
+		let wasInvoked = false;
+		const onCallbackInvoked = () => {
+			wasInvoked = true;
+			return true;
+		};
+		should(tiModule.callback).be.equalOneOf([ null, undefined ]);
+		tiModule.callback = onCallbackInvoked;
+		should(tiModule.callback).be.eql(onCallbackInvoked);
+		should(tiModule.callback()).be.true();
+		should(wasInvoked).be.true();
+	});
+
+	it('#invokeCallbackSync()', () => {
+		tiModule.callback = (arg1, arg2, arg3) => {
+			if ((arg1 === true) && (arg2 === 123) && (arg3 === 'test')) {
+				return 'succeeded';
+			}
+			return 'failed';
+		};
+		should(tiModule.invokeCallbackSync(true, 123, 'test')).be.eql('succeeded');
+	});
+
+	it('#invokeCallbackAsync()', function (finish) {
+		this.timeout(1000);
+		let wasInvoked = false;
+		tiModule.callback = (arg1, arg2, arg3) => {
+			try {
+				wasInvoked = true;
+				should(arg1).be.true();
+				should(arg2).be.eql(123);
+				should(arg3).be.eql('test');
+				finish();
+			} catch (err) {
+				finish(err);
+			}
+		};
+		tiModule.invokeCallbackAsync(true, 123, 'test');
+		should(wasInvoked).be.false();
+	});
+
+	it('Resource File', () => {
+		const jsonData = require('/ti.modulesdk920/test.json');
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.modulesdk920.test.js;
  Line: 251;
-->
- :no_entry_sign: tests/Resources/ti.modulesdk920.test.js line 251 – Do not import modules using an absolute path (import/no-absolute-path)